bdai_ros2_wrappers/bdai_ros2_wrappers/action.py

```diff
@@ -379,6 +383,7 @@ def synchronously(
         *,
         feedback_callback: Optional[Callable] = None,
         timeout_sec: Optional[float] = None,
+        nothrow: bool = False,
```

Comment:
Should `nothrow` be set here or in the constructor? The `result` property also has the chance of throwing

Reply:
> Should nothrow be set here or in the constructor?

It could be set on construction, though I'd rather not encourage the workflow. This is a escape hatch for existing code.

> The `result` property also has the chance of throwing

Yes. Same with the constructor. Same for `wait_for_future`. A strict `nothrow` would require a try-catch-all clause. This simply skips the nominal exceptions. We can make it strict. Existing APIs are not really explicit about their failure modes.

Reply:
If this fits the need then I think it is fine. No need to add extra infrastructure if we don't need it